docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md

```diff
@@ -345,38 +356,61 @@ The HttpDataAddress above configures the following behavior:
   backend
   `http://mydtr.com/api/v3/shell-descriptors`
 - `proxyQueryParams`: This string can be either "true" or "false". If set to true, the EDC Data Plane will always
-  forward
-  query parameters to the backend. For the /lookup APIs of the registry, this is critical.
+  forward query parameters to the backend. For the /lookup APIs of the registry, this is critical.
 - `proxyMethod`: This string can be either "true" or "false". If "false", the Data Plane will change the HTTP-Verb to
-  GET
-  for all incoming requests. As there is no scenario in the Catena-X scope where a business partner manipulates data in
-  a
-  foreign DTR, it should be set to false.
+  GET for all incoming requests. As there is no scenario in the Catena-X scope where a business partner manipulates
+  data in a foreign DTR, it should be set to false.
```

Comment:
The new AAS 3 includes the API lookup/shellsByAssetLink which is a post method. This method will be used to find ShellIds based on specificAssetIds like the GET /lookup. Because of that, the parameter needs to set to true.

Reply:
However, that's neither in the current IDTA-release of the AAS-standard nor in CX-0002. Providers and consumers still rely on the old `GET /lookup/shells?assetIds=foo` API. That's why this "sneak-preview" feature should (at this point in time) only be documented as implemenation features and not at in the kit, I think.